cmake/packaging/rpm.cmake

```diff
@@ -85,7 +88,74 @@ macro(ov_cpack_settings)
         # - 2022.1.0 is the last public release with rpm packages from Intel install team
         # - 2022.1.1 does not have rpm packages enabled, distributed only as archives
         2022.1.0)
+    
+    #
+    # Frontends
+    #
+
+    if(ENABLE_OV_IR_FRONTEND)
+        set(CPACK_COMPONENT_IR_DESCRIPTION "OpenVINO IR Frontend")
+        set(CPACK_COMPONENT_IR_DEPENDS "${OV_CPACK_COMP_CORE}")
+        set(CPACK_RPM_IR_PACKAGE_NAME "libopenvino-ir-frontend-${cpack_name_ver}")
+        set(CPACK_RPM_IR_PACKAGE_CONTROL_EXTRA "${def_postinst};${def_postrm}")
```

Comment:
BTW, I don't see such variables `_PACKAGE_CONTROL_EXTRA` in https://cmake.org/cmake/help/latest/cpack_gen/rpm.html